src/cpu/test.rs

```diff
@@ -1508,6 +1508,41 @@ fn test_ld_sp_hl() {
     assert_eq!(cpu.get_reg_16(Register16Bit::SP), 0x69);
 }
 
+// Verify loading contents of SP + s8 to HL
+#[test]
+fn test_ld_hl_sp_plus_s8() {
```

Comment:
Also test adding -1 to 0xffff (this should result in 0xfffe and both carry and half carry being set)